src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { Loader } from './Loader';
+import PersonLink from './PersonLink';
+import { useParams } from 'react-router-dom';
+
+interface PeopleProps {
+  person: Person[];
+  loading: boolean;
+}
+
+const People: React.FC<PeopleProps> = ({ person, loading }) => {
+  const { slug } = useParams<{ slug: string }>();
+  const [selectedPerson, setSelectedPerson] = useState(slug || '');
```

Comment:
why this part of code is required?

```
const [selectedPerson, setSelectedPerson] = useState(slug || '')

  useEffect(() => {
    if (slug) {
      setSelectedPerson(slug);
    }
  }, [slug]);
```

why not use the slug value directly? Or like:

`const selectedPerson = slug || ''`